src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
 
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
```

Comment:
The method `getSalaryInfo` is quite long and does a lot of things. This makes it harder to understand and maintain. You might want to consider breaking it down into smaller methods, each doing one specific thing. For example, you could have a method to calculate the salary for a single employee, and another method to generate the report.

Reply:
Try to follow these advices